tests/target-browser/test_helper.go

```diff
@@ -0,0 +1,210 @@
+
+	// unmounting from default target path
+	UnMountTarget()
+	log.Info("unmounted from default path")
+
+	time.Sleep(time.Second * 10)
+
+	// setting "NFS_SERVER_BASE_PATH" to newly formed directory and mounting to it
+	gomega.Expect(os.Setenv(NFSServerBasePath, TargetBrowserDataPath+"/"+randomDirectory)).To(gomega.BeNil())
+	MountTarget()
+	log.Info("mounted to new path")
+	gomega.Expect(err).To(gomega.BeNil())
+
+	time.Sleep(time.Second * 20)
+}
+
+func MountTarget() {
+	targetBrowserPath := os.Getenv(NFSServerBasePath)
+	c := fmt.Sprintf("sudo mount -t nfs -o  nfsvers=4 %s:%s %s", NfsIPAddress, targetBrowserPath, targetLocation)
+	fmt.Printf("Mount command %s\n", c)
+	command := exec.Command("bash", "-c", c)
+	_, err := command.CombinedOutput()
+	if err != nil {
+		log.Errorf("error %s", err.Error())
+	}
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+}
+
+func removeRandomDirAndUnmount() {
+
+	// unmounting from random named directory
+	UnMountTarget()
+	log.Info("unmounted from random named directory", randomDirectory)
+
+	time.Sleep(time.Second * 10)
+
+	// setting "NFS_SERVER_BASE_PATH" to default target path and mounting to it
+	gomega.Expect(os.Setenv(NFSServerBasePath, TargetBrowserDataPath)).To(gomega.BeNil())
+	MountTarget()
+	log.Info("mounted to default path")
+
+	time.Sleep(time.Second * 10)
+
+	// removing random named directory
+	_, _ = shell.RmRf(filepath.Join(targetLocation, randomDirectory))
+
+	log.Info("removed directory")
+}
+
+func verifyBackupPlansAndBackupsOnNFS(backupPlans, backups int) (backupPlanUIDs []string) {
+
+	var (
+		err                        error
+		tempBackupUIDs, backupUIDs []string
+	)
+
+	gomega.Eventually(func() []string {
+		return gomega.InterceptGomegaFailures(func() {
+			backupPlanUIDs, err = shell.ReadChildDir(targetLocation)
+			gomega.Expect(err).To(gomega.BeNil())
+			log.Info(len(backupPlanUIDs), " backupplans present on target location")
+			gomega.Expect(len(backupPlanUIDs)).To(gomega.Equal(backupPlans))
+		})
+	}, timeout, interval).Should(gomega.BeEmpty())
+
+	gomega.Eventually(func() []string {
+		return gomega.InterceptGomegaFailures(func() {
+			for i := range backupPlanUIDs {
+				tempBackupUIDs, err = shell.ReadChildDir(targetLocation + "/" + backupPlanUIDs[i])
+				gomega.Expect(err).To(gomega.BeNil())
+				backupUIDs = append(backupUIDs, tempBackupUIDs...)
+			}
+			log.Info(len(backupUIDs), " backups present on target location")
+			gomega.Expect(len(backupUIDs)).To(gomega.Equal(backups))
+		})
+	}, timeout, interval).Should(gomega.BeEmpty())
+
+	return backupPlanUIDs
+
+}
+
+func VerifyTargetStatus(installNs string) {
+	gomega.Eventually(func() bool {
+		getTarget := fmt.Sprintf("kubectl get target %s  --namespace %s -o=jsonpath=\"{.items[*]}{.status.status}\"",
+			TargetName, installNs)
+		cmd := exec.Command("bash", "-c", getTarget)
+		output, err := cmd.CombinedOutput()
+		if err != nil {
+			log.Errorf("Error to execute command %s", err.Error())
+		}
```

Comment:
you can use `client.Get` method with unstructed type.

Reply:
Fixed it